crates/biome_html_analyze/tests/specs/nursery/useControlLabel/html/valid.html

```
@@ -0,0 +1,33 @@
+<!-- should not generate diagnostics -->
+
+<button>Submit</button>
```

HTMLHint (1.9.2)

[error] Doctype must be declared before any non-comment content.

(doctype-first)